src/mercury_engine_data_structures/formats/brfld.py

```diff
@@ -20,8 +20,10 @@
 
 logger = logging.getLogger(__name__)
 
+ActorLink = str
```

Comment:
actually strictly speaking a link can be to anything in the brfld, right? maybe ScenarioLink or something would be more appropriate? not a huge deal tho and if you prefer ActorLink we'll use it